### PR TITLE
Linked fields

### PR DESCRIPTION
**Pull request by virginiacc**
### Additions
- Link icons & disabled second inputs for linked fields in scenarios
- Disabled dropdown background image
- `styled-text-input` component to add dollar & percentage signs on down payment & price

### Changes
- Break out `loan-input-table-cell` component from `loan-input-table-row`
- Minor revisions to input components to pass in `disabled` attribute

### Screenshots
screen shot 2015-06-02 at 8 10 12 pm

### Review
@cfarm 
@stephanieosan 
@huetingj

### CODE REVIEW

**Comment by huetingj** [comment]
:+1: sweet!

**Comment by cfarm** [comment]
I checked this out locally and I can't edit the down payment inputs when in the down payment scenario. Is this intended?

**Comment by cfarm** [comment]
Finding the same issue in the 'Design your own scenario' interface for down payment inputs

**Comment by virginiacc** [comment]
@cfarm Nope, not intentional. :) Should be fixed now...

File: src/static/css/module/loan-comparison.less [hunk 1]
```diff
@@ -751,11 +747,66 @@
...
+        }
+      }
     }
   
-    tbody tr {
+    tr {
       border: none;
+      
+      &.padded-row {
+        td.label-cell {
```
Comment (cfarm) [inline src/static/css/module/loan-comparison.less]: after testing we should revisit the nesting in this section and see if we can reduce it.

File: src/static/js/modules/loan-comparison/components/loan-input-table-row.js [hunk 2]
```diff
@@ -47,66 +12,73 @@ var LoanInputRow = React.createClass({
...
         scenario: React.PropTypes.object // or null
     },
-    handleChange: function (loanId, prop, eventOrVal) {
-        var val;
-        if (typeof eventOrVal === 'object' && eventOrVal) {
-            val = eventOrVal.target.value;
-        } else {
-            val = eventOrVal;
+    
+    generateClassName: function (rowType) {
```
Comment (cfarm) [inline src/static/js/modules/loan-comparison/components/loan-input-table-row.js]: I'm going to start using the same naming conventions as you so we keep these functions consistent between components.

**Comment by cfarm** [comment]
Yay it works great now :+1:

**Pull request merged**